__tests__/controllers/auth.controller.spec.ts

```diff
@@ -0,0 +1,58 @@
+import * as Faker from 'faker';
+import request from 'supertest';
+import app from '@app';
+import connection from '@database/connection';
+import { User } from '@entities/user.entity';
+
+const API = '/api/v1/auth';
+
+beforeAll(async () => {
+  await connection.create();
+});
+
+afterAll(async () => {
+  await connection.close();
+});
+
+beforeEach(async () => {
+  await connection.clear();
+});
+
+describe('when creating a sigup', () => {
+  it('should return 200 whith valid params', async () => {
```

Comment:
Just guessing but is it a good practice to use `should` instead of just putting `returns` as it's an expectation

Reply:
Same in the other tests